---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Genomic and transcriptomic resources for candidate gene discovery in the Ranunculids.
subtitle: ''
summary: ''
authors:
- Tatiana Arias
- Diego Mauricio Riaño-Pachón
- Verónica S. Di Stilio
tags:
- '"Ranunculaceae; Thalictrum hernandezii; Thalictrum thalictroides; draft genome;
  floral transcriptome; pollination syndrome; sexual system"'
categories: []
date: '2021-01-01'
lastmod: 2021-11-02T23:52:05-03:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-01-24T20:52:30.161366Z'
publication_types:
- '2'
abstract: Multiple transitions from insect to wind pollination are associated with
  polyploidy and unisexual flowers in   (Ranunculaceae), yet the underlying genetics
  remains unknown. We generated a draft genome of  , a representative of a clade with
  ancestral floral traits (diploid, hermaphrodite, and insect pollinated) and a model
  for functional studies. Floral transcriptomes of   and of wind-pollinated, andromonoecious   are
  presented as a resource to facilitate candidate gene discovery in flowers with different
  sexual and pollination systems. A draft genome of   and two floral transcriptomes
  of   and   were obtained from HiSeq 2000 Illumina sequencing and de novo assembly.
  The   de novo draft genome assembly consisted of 44,860 contigs (N50 = 12,761 bp,
  243 Mbp total length) and contained 84.5% conserved embryophyte single-copy genes.
  Floral transcriptomes contained representatives of most eukaryotic core genes, and
  most of their genes formed orthogroups. To validate the utility of these resources,
  potential candidate genes were identified for the different floral morphologies
  using stepwise data set comparisons. Single-copy gene analysis and simple sequence
  repeat markers were also generated as a resource for population-level and phylogenetic
  studies.
publication: '*Applications in plant sciences*'
doi: 10.1002/aps3.11407
---
