---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Gene Co-expression Network Reveals Potential New Genes Related to Sugarcane
  Bagasse Degradation in Trichoderma reesei RUT-30.
subtitle: ''
summary: ''
authors:
- Gustavo Pagotto Borin
- Marcelo Falsarella Carazzolle
- Renato Augusto Corrêa Dos Santos
- Diego Mauricio Riaño-Pachón
- Juliana Velasco de Castro Oliveira
tags:
- '"2G ethanol"'
- '"Trichoderma reesei"'
- '"Xyr1-binding site"'
- '"enzymatic cocktail"'
- '"gene co-expression network"'
- '"sugarcane bagasse"'
categories: []
date: '2018-01-01'
lastmod: 2021-11-03T00:16:18-03:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-11-04T01:46:01.053416Z'
publication_types:
- '2'
abstract: The biomass-degrading fungus Trichoderma reesei has been considered a model
  for cellulose degradation, and it is the primary source of the industrial enzymatic
  cocktails used in second-generation (2G) ethanol production. However, although various
  studies and advances have been conducted to understand the cellulolytic system and
  the transcriptional regulation of T. reesei, the whole set of genes related to lignocellulose
  degradation has not been completely elucidated. In this study, we inferred a weighted
  gene co-expression network analysis based on the transcriptome dataset of the T.
  reesei RUT-C30 strain aiming to identify new target genes involved in sugarcane
  bagasse breakdown. In total, ~70% of all the differentially expressed genes were
  found in 28 highly connected gene modules. Several cellulases, sugar transporters,
  and hypothetical proteins coding genes upregulated in bagasse were grouped into
  the same modules. Among them, a single module contained the most representative
  core of cellulolytic enzymes (cellobiohydrolase, endoglucanase, β-glucosidase, and
  lytic polysaccharide monooxygenase). In addition, functional analysis using Gene
  Ontology (GO) revealed various classes of hydrolytic activity, cellulase activity,
  carbohydrate binding and cation:sugar symporter activity enriched in these modules.
  Several modules also showed GO enrichment for transcription factor activity, indicating
  the presence of transcriptional regulators along with the genes involved in cellulose
  breakdown and sugar transport as well as other genes encoding proteins with unknown
  functions. Highly connected genes (hubs) were also identified within each module,
  such as predicted transcription factors and genes encoding hypothetical proteins.
  In addition, various hubs contained at least one DNA binding site for the master
  activator Xyr1 according to our in silico analysis. The prediction of Xyr1 binding
  sites and the co-expression with genes encoding carbohydrate active enzymes and
  sugar transporters suggest a putative role of these hubs in bagasse cell wall deconstruction.
  Our results demonstrate a vast range of new promising targets that merit additional
  studies to improve the cellulolytic potential of T. reesei strains and to decrease
  the production costs of 2G ethanol.
publication: '*Frontiers in bioengineering and biotechnology*'
---
