---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Proteome-wide survey of phosphorylation patterns affected by nuclear DNA polymorphisms
  in Arabidopsis thaliana.
subtitle: ''
summary: ''
authors:
- Diego Mauricio Riaño-Pachón
- Sabrina Kleessen
- Jost Neigenfind
- Pawel Durek
- Elke Weber
- Wolfgang R. Engelsberger
- Dirk Walther
- Joachim Selbig
- Waltraud X. Schulze
- Birgit Kersten
tags:
- '"Amino Acid Sequence"'
- '"Arabidopsis/cytology/*genetics/*metabolism"'
- '"Binding Sites"'
- '"Cell Nucleus/*genetics"'
- '"DNA"'
- '"Plant/*genetics"'
- '"Humans"'
- '"Molecular Sequence Data"'
- '"Phosphorylation"'
- '"Plant Proteins/chemistry/genetics/metabolism"'
- '"*Polymorphism"'
- '"Single Nucleotide"'
- '"Proteome/chemistry/genetics/*metabolism"'
categories: []
date: '2010-07-01'
lastmod: 2021-11-03T00:16:53-03:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-11-03T03:16:52.893410Z'
publication_types:
- '2'
abstract: 'BACKGROUND: Protein phosphorylation is an important post-translational
  modification influencing many aspects of dynamic cellular behavior. Site-specific
  phosphorylation of amino acid residues serine, threonine, and tyrosine can have
  profound effects on protein structure, activity, stability, and interaction with
  other biomolecules. Phosphorylation sites can be affected in diverse ways in members
  of any species, one such way is through single nucleotide polymorphisms (SNPs).
  The availability of large numbers of experimentally identified phosphorylation sites,
  and of natural variation datasets in Arabidopsis thaliana prompted us to analyze
  the effect of non-synonymous SNPs (nsSNPs) onto phosphorylation sites. RESULTS:
  From the analyses of 7,178 experimentally identified phosphorylation sites we found
  that: (i) Proteins with multiple phosphorylation sites occur more often than expected
  by chance. (ii) Phosphorylation hotspots show a preference to be located outside
  conserved domains. (iii) nsSNPs affected experimental phosphorylation sites as much
  as the corresponding non-phosphorylated amino acid residues. (iv) Losses of experimental
  phosphorylation sites by nsSNPs were identified in 86 A. thaliana proteins, among
  them receptor proteins were overrepresented.These results were confirmed by similar
  analyses of predicted phosphorylation sites in A. thaliana. In addition, predicted
  threonine phosphorylation sites showed a significant enrichment of nsSNPs towards
  asparagines and a significant depletion of the synonymous substitution. Proteins
  in which predicted phosphorylation sites were affected by nsSNPs (loss and gain),
  were determined to be mainly receptor proteins, stress response proteins and proteins
  involved in nucleotide and protein binding. Proteins involved in metabolism, catalytic
  activity and biosynthesis were less affected. CONCLUSIONS: We analyzed more than
  7,100 experimentally identified phosphorylation sites in almost 4,300 protein-coding
  loci in silico, thus constituting the largest phosphoproteomics dataset for A. thaliana
  available to date. Our findings suggest a relatively high variability in the presence
  or absence of phosphorylation sites between different natural accessions in receptor
  and other proteins involved in signal transduction. Elucidating the effect of phosphorylation
  sites affected by nsSNPs on adaptive responses represents an exciting research goal
  for the future.'
publication: '*BMC genomics*'
---
