---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Genome-wide analysis of ABA-responsive elements ABRE and CE3 reveals divergent
  patterns in Arabidopsis and rice.
subtitle: ''
summary: ''
authors:
- Judith L. Gómez-Porras
- Diego Mauricio Riaño-Pachón
- Ingo Dreyer
- Jorge E. Mayer
- Bernd Mueller-Roeber
tags:
- Abscisic Acid/*pharmacology
- Arabidopsis/*genetics
- Base Sequence
- Chromosome Mapping/methods
- Computational Biology
- Gene Dosage
- Gene Expression Regulation
- Plant/*drug effects
- Gene Order
- Genes
- Plant
- '*Genome'
- Molecular Sequence Data
- Oryza/*genetics
- Response Elements/*drug effects
categories: []
date: '2007-08-01'
lastmod: 2021-11-03T00:17:01-03:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-08-30T13:17:15.031700Z'
publication_types:
- '2'
abstract: 'BACKGROUND: In plants, complex regulatory mechanisms are at the core of
  physiological and developmental processes. The phytohormone abscisic acid (ABA)
  is involved in the regulation of various such processes, including stomatal closure,
  seed and bud dormancy, and physiological responses to cold, drought and salinity
  stress. The underlying tissue or plant-wide control circuits often include combinatorial
  gene regulatory mechanisms and networks that we are only beginning to unravel with
  the help of new molecular tools. The increasing availability of genomic sequences
  and gene expression data enables us to dissect ABA regulatory mechanisms at the
  individual gene expression level. In this paper we used an in-silico-based approach
  directed towards genome-wide prediction and identification of specific features
  of ABA-responsive elements. In particular we analysed the genome-wide occurrence
  and positional arrangements of two well-described ABA-responsive cis-regulatory
  elements (CREs), ABRE and CE3, in thale cress (Arabidopsis thaliana) and rice (Oryza
  sativa). RESULTS: Our results show that Arabidopsis and rice use the ABA-responsive
  elements ABRE and CE3 distinctively. Earlier reports for various monocots have identified
  CE3 as a coupling element (CE) associated with ABRE. Surprisingly, we found that
  while ABRE is equally abundant in both species, CE3 is practically absent in Arabidopsis.
  ABRE-ABRE pairs are common in both genomes, suggesting that these can form functional
  ABA-responsive complexes (ABRCs) in Arabidopsis and rice. Furthermore, we detected
  distinct combinations, orientation patterns and DNA strand preferences of ABRE and
  CE3 motifs in rice gene promoters. CONCLUSION: Our computational analyses revealed
  distinct recruitment patterns of ABA-responsive CREs in upstream sequences of Arabidopsis
  and rice. The apparent absence of CE3s in Arabidopsis suggests that another CE pairs
  with ABRE to establish a functional ABRC capable of interacting with transcription
  factors. Further studies will be needed to test whether the observed differences
  are extrapolatable to monocots and dicots in general, and to understand how they
  contribute to the fine-tuning of the hormonal response. The outcome of our investigation
  can now be used to direct future experimentation designed to further dissect the
  ABA-dependent regulatory networks.'
publication: '*BMC genomics*'
---
